lib/binary_to_decimal.rb

```diff
@@ -3,7 +3,18 @@
 # The most significant bit is at index 0.
 # The least significant bit is at index 7.
 # Calculate  and return the decimal value for this binary number using
-# the algorithm you devised in class.
+# the algorithm you devised in #class.
 def binary_to_decimal(binary_array)
```

Comment:
This definitely works, but can you see how to do it without the Enumerable method `reverse`.  Can you make a loop which counts down the index from `binary_array.length - 1` to 0 and solve it that way?